dandiapi/api/apps.py

```diff
@@ -50,13 +50,15 @@ def ready(self):
         import dandiapi.api.checks  # noqa: F401, RUF100
         import dandiapi.api.signals  # noqa: F401
 
+        logging.getLogger(__name__).setLevel(settings.DANDI_LOG_LEVEL)
+
         if hasattr(settings, 'SENTRY_DSN'):
             sentry_sdk.init(
                 dsn=settings.SENTRY_DSN,
                 environment=settings.SENTRY_ENVIRONMENT,
                 release=settings.SENTRY_RELEASE,
                 integrations=[
-                    LoggingIntegration(level=logging.INFO, event_level=logging.WARNING),
+                    LoggingIntegration(level=settings.DANDI_LOG_LEVEL, event_level=logging.WARNING),
```

Comment:
This should actually remain unchanged: we need for Sentry to receive `INFO` level events, as it uses those to create the breadcrumbs that make up its frame-level reporting for error events. See 74697eaf862d122153a1f29357c400cfb58ade1f.

Reply:
roger that, makes sense, thanks!